### ISSUE TITLE
Julia install error (new in 0.6.5)

### ISSUE THREAD

**Issue by yxie20** [MilesCranmer/PySR#58]
I'm getting this error with pysr>=0.6.5, during the first run of pysr after `pip install pysr`.
```
Running on julia -O0 /tmp/tmpy5eg_cxg/runfile.jl
  Activating environment at `~/anaconda3/envs/rw/lib/python3.7/site-packages/Project.toml`
    Updating registry at `~/.julia/registries/General`
ERROR: LoadError: Unsatisfiable requirements detected for package SymbolicRegression [8254be44]:
 SymbolicRegression [8254be44] log:
 ├─possible versions are: 0.1.0-0.6.4 or uninstalled
 └─restricted to versions 0.6.5-0.6 by an explicit requirement — no versions left
```

**Comment by MilesCranmer** [comment]
Can you try this fix? https://github.com/MilesCranmer/PySR/issues/27

Unfortunately the Julia servers go out of date quite frequently. It is really frustrating. The only solution is to have everybody switch to the git version of the Julia registry, rather than their web service one.

**Comment by MilesCranmer** [comment]
Fixed using #27 as mentioned in #53.
